### ISSUE TITLE
Show-hide transitions based on ng-show

### ISSUE THREAD

**Issue by ghost** [rajeshwarpatlolla/ionic-toast#15]
It would be great to have some default show and hide animations, i.e. fade-in and fade-out.

**Comment by sonicwong** [comment]
Yes, we should use "ng-show" / "ng-hide" instead of apply style(display:none) to DOM directly.
